### PR TITLE
DOC-538: Various content CSS page fixes

### PR DESCRIPTION
**Pull request by tylerkelly13**
Related Ticket: DOC-538

Description of Changes:
* Fix premium skins product name
* Add info about default CSS
* Fix boilerplate CSS content 

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] `_data/nav.yml` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

File: general-configuration-guide/boilerplate-content-css.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+---
+layout: default
+title: Boilerplate content CSS
+title_nav: Boilerplate content CSS
+description_short: Learn how to set up CSS for your site to integrate TinyMCE.
+description: Learn how to set up CSS for your site to integrate TinyMCE.
+keywords: css content_css
+---
+
+Some features in {{site.productname}} that require additional CSS to be added to the page that displays the content produced by the editor. In general, to get as clean an output as possible from a rich text editor as well as to use some advanced features like captioned images, use CSS rather than inline styles to render correctly.
```
Comment (lnewson) [inline general-configuration-guide/boilerplate-content-css.md:10]: `s/that require/require/`?

Comment (lnewson) [reply general-configuration-guide/boilerplate-content-css.md:10]: Second sentence is also very long. It feels like it needs a comma or should be broken up.

Comment (tylerkelly13) [reply general-configuration-guide/boilerplate-content-css.md:10]: that whole paragraph was horrible (I just brought it over as-is). Significant revisions made.

Comment (tylerkelly13) [reply general-configuration-guide/boilerplate-content-css.md]: May be fixed by https://github.com/tinymce/tinymce-docs/pull/1605/commits/64c3e93fc457c41ffb4b02d1213bcbb580b98674

File: _includes/css-codeblock/toc-plugin-css.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+/* Basic styles for Table of Contents plugin (toc) */
+.mce-toc {
+	border: 1px solid gray;
```
[the remark this answers is not in the document]
Comment (tylerkelly13) [reply _includes/css-codeblock/toc-plugin-css.md]: Fixed by https://github.com/tinymce/tinymce-docs/pull/1605/commits/64c3e93fc457c41ffb4b02d1213bcbb580b98674

File: _includes/css-codeblock/mediaembed-plugin-css.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
+.ephox-summary-card {
```
[the remark this answers is not in the document]
Comment (tylerkelly13) [reply _includes/css-codeblock/mediaembed-plugin-css.md:1]: Fixed by https://github.com/tinymce/tinymce-docs/pull/1605/commits/64c3e93fc457c41ffb4b02d1213bcbb580b98674

File: _includes/css-codeblock/image-plugin-css.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+figure.image {
+	display: inline-block;
```
[the remark this answers is not in the document]
Comment (tylerkelly13) [reply _includes/css-codeblock/image-plugin-css.md]: Fixed by https://github.com/tinymce/tinymce-docs/pull/1605/commits/64c3e93fc457c41ffb4b02d1213bcbb580b98674

File: _includes/configuration/shipped-content-css.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
+{{site.productname}} comes with four content CSS:
+
+- `default`
+- `dark`
+- `document`
+- `writer`
+
+These CSS can be added to the editor configuration using the `content_css` option. They can also be used as a template for creating a custom CSS for the editor. For the CSS source code, see: [{{site.productname}} GitHub Repository - Provided Content CSS](https://github.com/tinymce/tinymce/tree/develop/modules/oxide/src/less/skins/content).
```
[the remark this answers is not in the document]
Comment (tylerkelly13) [reply _includes/configuration/shipped-content-css.md]: May be fixed by https://github.com/tinymce/tinymce-docs/pull/1605/commits/64c3e93fc457c41ffb4b02d1213bcbb580b98674

[the remark this answers is not in the document]
Comment (lnewson) [reply _includes/configuration/shipped-content-css.md]: The first part is fixed, still not sure on linking to the less files though, so leaving this unresolved so others can comment.

File: _includes/css-codeblock/pageembed-plugin-css.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+.tiny-pageembed--1by1 {
+  display: block;
+  overflow: hidden;
+  padding: 0;
+  position: relative;
+  width: 100%;
+}
+
+.tiny-pageembed--21by9 {
+  padding-top: 42.857143%;
```
Comment (ashfordneil) [inline _includes/css-codeblock/pageembed-plugin-css.md:13]: Oddly specific

Comment (tylerkelly13) [reply _includes/css-codeblock/pageembed-plugin-css.md:13]: feature not a bug

Comment (tylerkelly13) [reply _includes/css-codeblock/pageembed-plugin-css.md:13]: 9 / 21~ 0.42857143

**Pull request merged**